package-lock.json

```diff
@@ -0,0 +1,10131 @@
+{
+  "name": "@quixo3/prisma-session-store",
+  "version": "3.1.0",
+  "lockfileVersion": 1,
```

Comment:
Are we moving away from `yarn`?

Reply:
Shoot - I didn't see that this had been included; sorry about that.

Agree it makes sense to move away from yarn, given that the npm cli now (I think?) has better capabilities for auditing/fixing security vulnerabilities, and is no longer significantly slower than yarn.

Going to remove package-lock in a follow-on PR, and save switching away from yarn / updating CI/CD for another day.

(Good catch!)

Reply:
👍🏻  `npm@7` is also faster than `yarn@1` and `yarn@2` has too many breaking changes with the PNP system to be a reasonable upgrade path.

I agree though that this can wait.

Reply:
>> npm@7 is also faster than yarn@1 and yarn@2 has too many breaking changes with the PNP system to be a reasonable upgrade path.

Good to know!

Reply:
@wSedlacek FYI, I just had to regenerate the token for CI/CD due to its outdated format; let me know if this causes you any trouble for you...